src/frontend/tsx/Challenges.tsx

```diff
@@ -186,8 +164,10 @@ export function Challenges () {
               className={(badChallengeAttempts > 0) ? 'rejected' : ''}
               type="text"
               placeholder={'~sampel-palnet'}
+              value={who}
               onChange={(e) => setWho(e.target.value)}
-              key={badChallengeAttempts}/>
+              key={badChallengeAttempts}
+              disabled={ challengingFriend ? true : false }/>
```

Comment:
`challengingFriend` is already a boolean, don't need to use the ternary operator